adapters/core2p2p/block.go

```diff
@@ -45,16 +45,15 @@ func AdaptHeader(header *core.Header, commitments *core.BlockCommitments,
 			NLeaves: header.EventCount,
 			Root:    AdaptHash(commitments.EventCommitment),
 		},
-		// todo fill receipts with receipt commitment
-		Receipts:        AdaptHash(&felt.Zero),
+		Receipts:        AdaptHash(commitments.ReceiptCommitment),
 		ProtocolVersion: header.ProtocolVersion,
-		GasPriceFri:     AdaptUint128(header.GasPrice),
+		GasPriceFri:     AdaptUint128(header.GasPriceSTRK),
 		Signatures:      utils.Map(header.Signatures, AdaptSignature),
 		StateDiffCommitment: &spec.StateDiffCommitment{
 			StateDiffLength: stateDiffLength,
-			Root:            AdaptHash(stateDiffCommitment),
+			Root:            AdaptHash(stateDiffCommitment), // todo commitments
```

Comment:
I want to debug why it works for pathfinder if we pass `commitments.StateDiffCommitment` here

Reply:
I passed incorrect value initially it should be not `stateDiff.Commitment()`, but `stateDiff.Hash()`